crates/bevy_asset/src/asset_server.rs

```diff
@@ -633,15 +668,128 @@ mod test {
 
     #[test]
     fn filename_with_dots() {
-        let asset_server = setup();
+        let asset_server = setup(".");
         let t = asset_server.get_path_asset_loader("test-v1.2.3.png");
         assert_eq!(t.unwrap().extensions()[0], "png");
     }
 
     #[test]
     fn multiple_extensions() {
-        let asset_server = setup();
+        let asset_server = setup(".");
         let t = asset_server.get_path_asset_loader("test.test.png");
         assert_eq!(t.unwrap().extensions()[0], "test.png");
     }
+
+    fn create_dir_and_file(file: impl AsRef<Path>) -> TempDir {
```

Comment:
Should we also have cleanup logic for this in our tests to avoid littering?

Reply:
Ah I see. The tempfile crate handles this. Can you remove the glob import so this is clearer?